Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/StatusBarHiddenMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	public enum StatusBarHiddenMode
+	{
+		Default,
+		True,
+		False
```

Comment:
i like this API more so user understands that Default is default .. and not null ..